app/views/datasets/show.html.haml

```diff
@@ -74,7 +74,7 @@
           - if d.artifact.current_path
             %small
               |
-              = link_to 'Download',
+              = link_to "Download (#{number_to_human_size(d.artifact.size.to_i)})",
```

Comment:
cool, never knew about `number_to_humanize_size`

Reply:
There are plenty of these things lurking in Rails